### PR TITLE
Add documentation for livechat setCustomField overwrite param

### PR DESCRIPTION
**Pull request by sampaiodiego**
Related to https://github.com/RocketChat/Rocket.Chat/pull/6036

### CODE REVIEW

File: 6. Developer Guides/Livechat API/README.md [hunk 1]
```diff
@@ -6,32 +6,30 @@ order: 5
...
-<script type="text/javascript">
-    RocketChat(function() {
-        this.setCustomField('fieldName1', 'Any value you want to store');
-        this.setCustomField('fieldName2', 'Any value you want to store');
-    });
-</script>
+```javascript
+RocketChat(function() {
+    this.setCustomField('fieldName1', 'Any value you want to store');
+    this.setCustomField('fieldName2', 'A value set just once', false); // you can pass false as the third parameter to no overwrite an already set value
```
Comment (graywolf336) [inline 6. Developer Guides/Livechat API/README.md]: Change `no` to `not`

**Review by graywolf336 — CHANGES_REQUESTED** [review]

File: 6. Developer Guides/Livechat API/README.md (hunk 1 above)
Comment (sampaiodiego) [reply 6. Developer Guides/Livechat API/README.md]: done :)

**Review by graywolf336 — APPROVED** [review]

**Comment by graywolf336** [comment]
@sampaiodiego You should also make note of which version of Rocket.Chat supports that.

**Comment by sampaiodiego** [comment]
true. I'll add that

.Diego Sampaio
+55 51 99252-6690

2017-02-15 16:24 GMT-02:00 Bradley Hilton <notifications@github.com>:

**Pull request merged**